### DISCUSSION TITLE
hi I need a software engineer

### DISCUSSION THREAD

**Discussion by GarGamelcom** [community/community#126813] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I want to implement a project that will require the help of good specialists

**Comment by ebndev** [comment]
Hi @GarGamelcom, thanks for being a part of the GitHub Community!

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
